azure-iot-device/tests/common/pipeline/pipeline_data_object_test.py

```diff
@@ -0,0 +1,134 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+
+fake_count = 0
```

Comment:
Does this ever get reset to 0?

I see it gets incremented in `get_next_fake_value()`, but I don't see the reset to 0

Reply:
Doesn't reset.  Doesn't need to reset.  I just wanted a unique auto-generated string that was human-readable.  

---
In reply to: [291658380](https://github.com/Azure/azure-iot-sdk-python-preview/pull/104#discussion_r291658380) [](ancestors = 291658380)